cpp/daal/src/algorithms/dtrees/gbt/gbt_train_dense_default_impl.i

```diff
@@ -317,7 +318,7 @@ void TrainBatchTaskBase<algorithmFPType, BinIndexType, cpu>::updateOOB(size_t iT
         auto pNode = dtrees::prediction::internal::findNode<algorithmFPType, TreeType, cpu>(t, x.get());
         DAAL_ASSERT(pNode);
         algorithmFPType inc = TreeType::NodeType::castLeaf(pNode)->response;
-        pf[iRow * _nTrees + iTree] += inc;
+        pf[iRow * _nTrees + iTree] += inc - res;
```

Comment:
I think clarification comment should be added to justify value decreasing.
e.g. " 'pf' buffer was already initialized by '_initialF' before first iteration"